### ISSUE TITLE
Is spwn_mult hardcoded?

### ISSUE THREAD

**Issue by Blevy2** [pdolder/MixFishSim#8]
Hi Paul,

I was working on getting more of the population to move to the spawning grounds during spawning weeks and noticed that regardless of what is entered for spwn_mult in a create_hab call, the value is fixed at 10 in create_hab. 

I am not sure why this is happening because even though 10 is set as the default in create_hab, it should be able to be overwritten. One can instead fix the value of spwn_mult inside create_hab as a work around.

best,

Ben Levy

**Comment by pdolder** [comment]
Hi Ben,

Is it this bit here:
https://github.com/pdolder/MixFishSim/blob/f4be072ef8328845a635e68d69a290be4b944fa5/R/create_hab.R#L104

?

I can see mult input is fixed to 2. That's maybe the problem?

Paul

**Comment by Blevy2** [comment]
Hi Paul,

I noticed that spwn_mult was fixed as 2 there, but I think that is the problem. Correct me if I am wrong but I don't see anywhere that spwn_loc is used in the package so I do not think this has an impact.

I had previously made my own version of create_hab to define strata within a domain for stratified random sampling. When I add spwn_mult <- 20 to the top of my create_hab it seems to override the hardcode and create a different spawning habitat. 

Ben

**Comment by Blevy2** [comment]
To clarify further, I added print(spwn_mult) to the top of my create_hab code and regardless of what I entered for spwn_mult (ex, create_hab(spwn_mult = 100) it was printing out spwn_mult as 10

Ben

**Comment by pdolder** [comment]
Ok, thanks. It's strange then that the default is not being overwritten by
the user input. I'll have to take a closer look at it.

Paul

**Comment by Blevy2** [comment]
I agree. I am pretty confused by this. Ill let you know if I figure it out.

Ben

**Comment by Blevy2** [comment]
Hi Paul,

Chris Legault helped me figure out what was happening here. There is no error in the source code, I just did not notice a misplaced parenthesis in Simple_MixFishSim_Example.Rmd in the create_hab call [the lines 87-95](https://github.com/pdolder/MixFishSim/blob/f4be072ef8328845a635e68d69a290be4b944fa5/vignettes/Simple_MixFishSim_Example.Rmd#L87-L95).

spawn_mult is inside of the spawn_areas list so create_hab does not overwrite the default of 10. Nothing big here. Sorry I did not notice that myself.

Ben
